### PR TITLE
Add spheron gpt

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "chalk": "^4.1.2",
     "cli-progress": "^3.12.0",
     "commonjs-ora": "^1.0.2",
-    "@spheron/core": "^1.0.0",
+    "@spheron/core": "file:../core",
```
Comment (dusanstanis) [inline packages/cli/package.json:35]: put it to ^1.0.7

File: packages/cli/src/configuration.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import path from "path";
 
 const configuration = {
-  spheronServerAddress: "https://api-v2.spheron.network",
-  spheronFrontendAddress: "https://app.spheron.network",
-  version: "1.0.11",
+  spheronServerAddress: "https://api-dev.spheron.network",
```
Comment (dusanstanis) [inline packages/cli/src/configuration.ts:4]: change this back to api-v2

File: packages/cli/src/configuration.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+  spheronFrontendAddress: "https://dev.spheron.network",
```
Comment (dusanstanis) [inline packages/cli/src/configuration.ts:5]: revert this as well

File: packages/cli/src/services/spheron-api.ts [hunk 4]
```diff
@@ -145,6 +147,22 @@ const SpheronApiService = {
       await client.getDeploymentEnvironments(projectId);
     return deploymentEnvironments;
   },
+
+  async generateCode(
+    spinner: Spinner,
```
Comment (dusanstanis) [inline packages/cli/src/services/spheron-api.ts:152]: why is spinner sent here ?

Comment (dusanstanis) [reply packages/cli/src/services/spheron-api.ts:152]: this should be done in command part of code. This code is only for calling client functions in my opinion

File: packages/cli/src/utils.ts [hunk 5]
```diff
@@ -91,3 +93,41 @@ export enum FileTypeEnum {
...
+  const directoryPath = path.join(rootDirectory, dir);
+  if (!fs.existsSync(directoryPath)) {
+    fs.mkdirSync(directoryPath, { recursive: true });
+  }
+
+  const fileFullPath = path.join(rootDirectory, filePath);
+  return fileFullPath;
+};
+
+// export const gptConfiguration = async (prompt: string) => {
```
Comment (dusanstanis) [inline packages/cli/src/utils.ts:111]: remove comments

**Review by dusanstanis — DISMISSED** [review]
Do change stuff that I commented out and it's approved.

File: packages/cli/src/services/spheron-api.ts (hunk 4 above)
Comment (aayushmahapatra) [reply packages/cli/src/services/spheron-api.ts:152]: So that we start the spinner only if the user is authorized.

**Review by Man-Jain — APPROVED** [review]

**Review by dusanstanis — APPROVED** [review]

**Pull request merged**